_posts/2022-03-28-embed-google-webfonts.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Embedding Google Fonts for better Privacy and GDPR conformance
+author: esko.suomi
+excerpt: >
+  Regional court in Munich, Germany recently rules Google Fonts as non-GDPR compliant due to the serving mechanism. This does pose a minor challenge, how does one embed webfonts to stay GDPR compliant?
+
+tags:
+ - Google Fonts
+ - GDPR
+ - Privacy
+---
+
+Recently the regional court in Germany in Munich declared that embedding Google Fonts on a website violates GDPR ([news article](https://rewis.io/urteile/urteil/lhm-20-01-2022-3-o-1749320/), [judgement](https://rewis.io/s/u/zH2/)).
+
+As a ripple effect, this judgement is now spreading through European Union member nations and especially public sector projects should take heed and replace the embedded fonts with self-hosted versions - after all, the issue isn't the fonts themselves, just the method of serving from an untrustworthy source.
```

Comment:
Grammarly would like a comma here, so would I :)

... through European Union member nations, and especially public sector ...